test/test_cli.py

```diff
@@ -0,0 +1,140 @@
+from os.path import realpath
+from pathlib import Path
+from unittest import TestCase
+import biggerquery as bgq
+from biggerquery.cli import *
+
+TEST_MODULE_SRC = 'test_module'
+
+
+class CacheBuilderTestCase(TestCase):
+
+    def test_walk_module_files(self):
```

Comment:
I suggest more descriptive test names, in this case maybe "test_should_walk_through_all_modules_inside_package_tree"

Reply:
I can change it this way, but for me it is easier to understand test when the name of the tested method is included, and mosts tests here are each designed to test one single method